server/src/internalClusterTest/java/org/elasticsearch/snapshots/SnapshotStressTestsIT.java

```diff
@@ -210,7 +210,7 @@ private static void logAndFailTest(Exception e) {
         private final ThreadPool threadPool = new TestThreadPool(
             "TrackedCluster",
             // a single thread for "client" activities, to limit the number of activities all starting at once
-            new ScalingExecutorBuilder(CLIENT, 1, 1, TimeValue.ZERO, CLIENT)
+            new ScalingExecutorBuilder(CLIENT, 1, 1, TimeValue.ZERO, true, CLIENT)
```

Comment:
Please, can we import static constant here and use **ZERO** instead **TimeValue.ZERO**? The code will be cleaner. But its just my opinion.

Reply:
I prefer to keep it the way it is :)